src/main/java/flaxbeard/steamcraft/client/render/BlockRuptureDiscRenderer.java

```diff
@@ -86,6 +86,8 @@ private void setBoundsMeta(Block block, float x, float y, float z, float x2, flo
             case 3:
                 block.setBlockBounds(x, y, z, x2, y2, z2);
                 break;
+            default: 
+            	break;
```

Comment:
Strange. In my IDE, "break"  statement is one space ahead of shown here: that is how it is required to be. In the classes in the upper section on this  page, they look alright. But in my IDE, they are all the same. How come   it is seen different  here, I am really confused.